mqtt.js

```diff
@@ -0,0 +1,69 @@
+import { connect } from 'mqtt'
+import { connectionOptions } from "./util";
+
+let client = undefined
+let callbacks = {}
+
+const match = (filter, topic) => {
+    const filterArr = filter.split('/')
+    const topicArr = topic.split('/')
+
+    for (let i = 0; i < filterArr.length; i++) {
+        if (filterArr[i] === '#') return filterArr.length - 1 <= topicArr.length
+        if (filterArr[i] !== '+' && filterArr[i] !== topicArr[i]) return false 
+    }
+    return filterArr.length === topicArr.length
+}
+
+export const MQTTConnect = (onConnectCallback) => {
+    client = connect(connectionOptions.mqttOpts)
+    if (typeof onConnectCallback === 'function') {
+        client.on('connect', onConnectCallback)
+    }
+    client.on('message', function (topic, payload) {
+        let callback = callbacks[topic]
+        let jsonPayload = JSON.stringify(payload.toString())
```

Comment:
stringify would convert this to JSON. I think you want to convert the JSON to an object using `JSON.parse`